locopy/database.py

```diff
@@ -19,10 +19,13 @@
 import time
 
 from .errors import CredentialsError, DBError
-from .logger import logger
+from .logger import get_logger, INFO
 from .utility import read_config_yaml
 
 
+logger = get_logger(__name__, INFO)
```

Comment:
I'm curious about this. Would this be overwriting user settings?

Reply:
So this would enable `INFO` level for the `database` module.
I think once you import and change the logging session to override it should be OK?
Let me test it out.

Reply:
So it seems to work that way. Doing this changes the s3 logging output: 

```
import snowflake.connector
import locopy
import logging

logger = logging.getLogger("locopy.s3")
logger.setLevel(logging.ERROR)
```

```
WARNING|2020-01-17 14:24:38,405|locopy.snowflake|snowflake.py|__init__|178: S3 credentials were not found. S3 functionality is disabled
WARNING|2020-01-17 14:24:38,405|locopy.snowflake|snowflake.py|__init__|179: Only internal stages are available
```

It drops the following:

```
INFO|2020-01-17 14:23:20,749|locopy.s3|s3.py|_set_session|101: Initialized AWS session.
```

Reply:
Ok, awesome.